cmd/net-state/main.go

```diff
@@ -0,0 +1,48 @@
+package main
```

Comment:
we should probably add a copyright line at the top of these. see https://github.com/storj/storj/blob/master/pkg/ranger/common.go